objects/file/definition.json

```diff
@@ -108,9 +108,9 @@
       "recommended": false
     },
     "malware-sample": {
-      "description": "The file itself (binary)",
-      "ui-priority": 1,
-      "misp-attribute": "malware-sample"
```

Comment:
malware-sample is definitely correct here, and working: https://github.com/MISP/PyMISP/blob/master/tests/testlive_comprehensive.py#L995

How are you using the object?

Reply:
I was trying to import a text file.
if I place the file as a non malicious plain attachment, it does not render it as an object, which cannot be refrenced with other objects

Reply:
Did you use the API, or the web interface?